apps/server/src/services/ElasticSearchClient.test.ts

```diff
@@ -607,12 +611,101 @@ describe("getRecommendedInstitutions", () => {
       },
     );
 
-    const recommendedInstitutions = await getRecommendedInstitutions(
-      MappedJobTypes.AGGREGATE,
-    );
+    const recommendedInstitutions = await getRecommendedInstitutions({
+      jobType: MappedJobTypes.AGGREGATE,
+    });
 
     expect(recommendedInstitutions).toEqual([]);
   });
+
+  it("filters out test institutions if filterTestBanks is true", async () => {
+    const mockPreferences: preferences.Preferences = {
+      ...testPreferences,
+    } as any;
+
+    jest
+      .spyOn(preferences, "getPreferences")
+      .mockResolvedValue(mockPreferences);
+
+    ElasticSearchMock.clearAll();
+
+    ElasticSearchMock.add(
+      {
+        method: "POST",
+        path: "/_mget",
+        body: {
+          docs: mockPreferences.recommendedInstitutions.map(
+            (institutionId: string) => ({
+              _index: "institutions",
+              _id: institutionId,
+            }),
+          ),
+        },
+      },
+      () => {
+        return {
+          docs: [
+            { _source: elasticSearchInstitutionDataFavs[0] },
+            { _source: elasticSearchInstitutionDataFavs[1] },
+            { _source: elasticSearchInstitutionDataFavs[2] },
+          ],
+        };
+      },
+    );
+
+    const recommendedInstitutions = await getRecommendedInstitutions({
+      jobType: MappedJobTypes.AGGREGATE,
+      filterTestBanks: true,
+    });
+
+    expect(recommendedInstitutions.length).toEqual(1);
+    expect(recommendedInstitutions).toEqual([
+      elasticSearchInstitutionDataFavs[0],
+    ]);
+  });
+
+  it("does not filter out test institutions if filterTestBanks is false", async () => {
+    const mockPreferences: preferences.Preferences = {
```

Comment:
There's a lot of duplication between these two tests. I'd probably abstract the setup or combine the tests.